charts/lighthouse/values.yaml

```diff
@@ -275,6 +275,9 @@ tektoncontroller:
   # tektoncontroller.terminationGracePeriodSeconds -- Termination grace period for tekton controller pods
   terminationGracePeriodSeconds: 180
 
+  # tektoncontroller.rerunPrReconciler -- Enables the rerun pipelinerun reconciler to run and update rerun pipelinerun status to github
```

Comment:
The usage message is cryptic if you don't happen to now the internals. Please change to something more user friendly. Also add the information to the README of the chart.